pkg/datapath/fake/cells.go

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package fake
+
+import (
+	"time"
+
+	"github.com/cilium/cilium/pkg/datapath/tables"
+	"github.com/cilium/cilium/pkg/datapath/types"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/maps/authmap"
+	"github.com/cilium/cilium/pkg/maps/egressmap"
+	"github.com/cilium/cilium/pkg/maps/signalmap"
+
+	fakeauthmap "github.com/cilium/cilium/pkg/maps/authmap/fake"
+	fakesignalmap "github.com/cilium/cilium/pkg/maps/signalmap/fake"
+)
+
+// Cell provides a fake version of the datapath cell.
+//
+// Used in integration tests such as test/controlplane.
+var Cell = cell.Module(
+	"fake-datapath",
+	"Fake Datapath",
+
+	cell.Provide(
```

Comment:
i wonder whether it would be worth to further extract a "fake-maps" cell.

but i think as long as the fake-datapath is small it seems ok as is.